.github/workflows/ci.yml

```
@@ -99,3 +99,17 @@ jobs:
         run: npm ci
       - name: Run ${{ matrix.name }}
         run: ${{ matrix.command }}
+
+  check:  # This job does nothing and is only used for the branch protection
+    if: always()
+
+    needs:
+      - test
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Decide whether the needed jobs succeeded or failed
+        uses: re-actors/alls-green@release/v1
```

Optionally, pin for stability/security:
```suggestion
        uses: re-actors/alls-green@198badcb65a1a44528f27d5da555c4be9f12eac6
```